Dockerfile

```diff
@@ -21,7 +21,7 @@ FROM builder as tester
 CMD ["make", "test"]
 
 FROM gcr.io/distroless/base-debian10 as vvgo
-COPY ./infra/vvgo/etc/mime.types /etc/
+COPY ./etc/mime.types /etc/
```

Comment:
Can we move this to `./data` in the repo?

Reply:
:ok: